pandas/plotting/_matplotlib/timeseries.py

```diff
@@ -165,6 +166,16 @@ def _get_ax_freq(ax):
     return ax_freq
 
 
+def get_period_alias(freq) -> Optional[str]:
```

Comment:
@datapythonista suggestions for a docstring here?  i dont know off the top of my head what this alias is used for.